eng/pipelines/jobs/prepare-signed-artifacts-PR.yml

```diff
@@ -15,19 +15,6 @@ jobs:
     clean: all
 
   steps:
-
```

Comment:
I don't think this template is referenced?

Reply:
Whoops, thanks for the catch!